lua/entities/gmod_wire_expression2/core/custom/cfc_methods.lua

```diff
@@ -103,3 +103,25 @@ e2function number maxLoss()
 
     return max
 end
+
+e2function array getAlive()
+    local arrayOfLiving = {}
+    local humans = player.GetHumans()
+
+    for _, human in pairs( humans ) do
+        if human:Alive() then arrayOfLiving[#arrayOfLiving + 1] = human end
```

Comment:
Can you use the `table.insert` method instead, #ARRAY tends to give false readings. In this case, it shouldn't be an issue but better safe than sorry.

Reply:
Also, I'm not that big of a fan of the one line if statement in this case... Maybe @brandonsturgeon  could give a better way of doing it right.

Reply:
#array is fine it only gives "false readings" if you don't know what you're doing and you use it for non sequential arrays. in cases were you are constructing the array yourself i don't see a problem with using it.

Reply:
what about the one line if statement bleck is iffy about?

Reply:
the question is which one is more efficient

Reply:
I'm not sure if it's feasible to try to test that.

Reply:
Definitely go with the `insert`

Reply:
what about how bleck said he was iffy about the one liner

Reply:
I just thought it looked ugly that's all...

Reply:
```lua
        if human:Alive() then table.insert( arrayofLiving, human) end
```
Looks okay to me.

One could make the argument that this was acceptable, too:
```lua
        local function addHuman( human)
            table.insert( arrayOfLiving, human )
        end

        if human:Alive() then addHuman( human ) end
```